module.modulemap

```diff
@@ -0,0 +1,5 @@
+
+module CLibzmq [system] {
+    header "zmq_swift.h"
```

Comment:
I think this can be `/usr/local/include/zmq.h`, and the zmq_swift.h file is not needed.